src/libunwind/libunwind.rs

```diff
@@ -1,7 +1,26 @@
 #![allow(nonstandard_style)]
 
+#[cfg(not(target_os = "hermit"))]
 use libc::{c_int, c_void, uintptr_t};
 
+#[cfg(target_os = "hermit")]
+pub type c_int = i32;
+
+#[cfg(target_os = "hermit")]
+pub type uintptr_t = usize;
+
+#[cfg(target_os = "hermit")]
+#[repr(u8)]
+#[allow(missing_copy_implementations)]
+#[allow(missing_debug_implementations)]
+pub enum c_void {
+    // Two dummy variants so the #[repr] attribute can be used.
+    #[doc(hidden)]
+    __variant1,
+    #[doc(hidden)]
+    __variant2,
+}
```

Comment:
Would it be possible to move these implementations to `libc`?

Reply:
It is already in libc. I forgot only to remove my workaround. I will fix it.